checks.d/kubernetes.py

```diff
@@ -233,3 +238,27 @@ def _update_metrics(self, instance, kube_settings):
             except Exception, e:
                 self.log.error("Unable to collect metrics for container: {0} ({1}".format(
                     subcontainer.get('name'), e))
+
+        pods_list = self._retrieve_pods_list()
+        self._update_pods_metrics(instance, pods_list)
+
+    def _update_pods_metrics(self, instance, pods):
+        controllers_map = defaultdict(list)
+        for pod in pods['items']:
+            node_name = pod['spec']['nodeName']
+            try:
+                created_by = json.loads(pod['metadata']['annotations']['kubernetes.io/created-by'])
+                if created_by['reference']['kind'] == 'ReplicationController':
+                    controllers_map[created_by['reference']['name']].append(node_name)
+            except KeyError:
+                continue
+
+        tags = instance.get('tags', [])
+        for ctrl, pods in controllers_map.iteritems():
+            _tags = tags[:]  # copy base tags
+            _tags.append('kube_replication_controller:{0}'.format(ctrl))
+            # at the moment kubelet api reports data only for the current node,
+            # we expect exactly one item in the set.
+            for node_name in set(pods):
+                _tags.append('node_name:{0}'.format(node_name))
```

Comment:
There is no need to tag by node_name. As you mentioned, the kubelet will just return the data for the current node, and we already have the host name passed automatically so you don't have to add this tag.